v2/internal/controllers/crd_storage_storageaccount_test.go

```diff
@@ -244,3 +214,94 @@ func StorageAccount_SecretsWrittenToDifferentKubeSecrets(tc *testcommon.KubePerT
 	tc.ExpectSecretHasKeys(webSecret, "web")
 	tc.ExpectSecretHasKeys(dfsSecret, "dfs")
 }
+
+func Test_Storage_StorageAccount_ManagementPolicies(t *testing.T) {
```

Comment:
I don't know that you need to actually add a new top level test here, since we already have a top level test creating a storage account.

Could you just make your `StorageAccount_ManagementPolicy_CRUD` a subtest of the top level storage account test on line 57?